packages/core/src/external/carequality/ihe-gateway-v2/outbound/ihe-gateway-v2-logic.ts

```diff
@@ -151,19 +161,35 @@ export async function createSignSendProcessXCPDRequest({
       cxId,
       index,
     });
-    try {
-      // TODO not sure if we should retry on timeout
-      await executeWithNetworkRetries(async () => axios.post(pdResponseUrl, result), {
-        initialDelay: 100,
-        maxAttempts: 5,
-        httpStatusCodesToRetry: [502, 504],
-        log,
-      });
-    } catch (error) {
-      const msg = "Failed to send PD response to internal CQ endpoint";
-      const extra = { cxId, patientId, result };
-      log(`${msg} - ${errorToString(error)} - ${JSON.stringify(extra)}`);
-      capture.error(msg, { extra: { ...extra, error } });
+    if (result.patientMatch) {
+      try {
+        // TODO not sure if we should retry on timeout
+        await executeWithNetworkRetries(async () => axios.post(pdResponseUrl, result), {
+          initialDelay: 100,
+          maxAttempts: 5,
+          httpStatusCodesToRetry: [502, 504],
+          log,
+        });
+      } catch (error) {
+        const msg = "Failed to send PD response to internal CQ endpoint";
+        const extra = { cxId, patientId, result };
+        log(`${msg} - ${errorToString(error)} - ${JSON.stringify(extra)}`);
+        capture.error(msg, { extra: { ...extra, error } });
+      }
+    } else {
+      try {
+        const filePath = `cxId=${cxId}/patientId=${patientId}/requestId=${result.id}/gwId=${result.gateway.id}/result.json`;
```

Comment:
Can we use or apply the same pattern as the functions `createDocumentFilePath` or `getFileName`, please?

Also, why the `cxId=` and others on the file path?

Lastly, id doesn't seem we need the last level, gateway ID, as it will have a single file?

Reply:
1. Path name is based on using Hive partitioning for future analytics -- not critical but more ideal unless it's important for the paths to match.

2. This is called per-gateway, if we don't have the last level the files will overwrite themselves. We can also have the file name be the gateway id, but this isn't helpful for future querying.